org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/QuickFixCodeActionServiceTest.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.ide.tests.server;
+
+import org.eclipse.xtext.ide.server.ServerModule;
+import org.eclipse.xtext.ide.server.concurrent.RequestManager;
+import org.eclipse.xtext.util.Modules2;
+import org.eclipse.xtext.xbase.lib.Procedures.Procedure1;
+import org.junit.Test;
+
+import com.google.inject.Module;
+
+/**
+ * @author Heinrich Weichert
+ */
+public class QuickFixCodeActionServiceTest extends AbstractTestLangLanguageServerTest {
```

Comment:
i assume this class can be dropped/is redundant now?

Reply:
I basically tests the same stuff, so yes :D